fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuEnhancedTrackingProtectionRobot.kt

```diff
@@ -66,6 +69,36 @@ class SettingsSubMenuEnhancedTrackingProtectionRobot {
         ),
     ).click()
 
+    fun scrollToGCPSettings(): ViewInteraction = onView(withId(R.id.recycler_view)).perform(
+        RecyclerViewActions.scrollTo<RecyclerView.ViewHolder>(
+            hasDescendant(withText(globalPrivacyControlSwitchText)),
+        ),
+    )
+    fun verifyGPCTextWithSwitchWidget() {
+        onView(
+            allOf(
+                withParentIndex(1),
```

Comment:
Only one small change: we usually don't use indexes as they can change easily. I think in this case, just the text selectors should be enough.